vscode/package.json

```diff
@@ -520,6 +525,12 @@
           "description": "Generative AI Key",
           "scope": "window"
         },
+        "konveyor.kai.demoMode": {
+          "type": "boolean",
+          "default": false,
+          "description": "Demo mode for Kai",
+          "scope": "window"
+        },
```

Comment:
Setting "konveyor.kai.demoMode" in settings.json did work, it just wasn't listed so it wasn't a documented config key.

Here is the previous discussion: https://github.com/konveyor/editor-extensions/pull/126#discussion_r1867961970

Reply:
Closing RE: solution discovered on slack.